tasks/interactive-bots.md

```diff
@@ -157,7 +166,10 @@ messages to users.
   The bot now responds to `@link` and `@resource`.
 
 
-* Edit the `handle_message`:
+* Create a `links` stream.
```

Comment:
@trueskawka should I just change the infinite loop aspect, meaning from "links" to "followup" stream?